data-prepper-plugins/kafka-plugins/build.gradle

```diff
@@ -9,12 +9,13 @@ plugins {
 
 dependencies {
     implementation project(':data-prepper-api')
-    implementation project(':data-prepper-plugins:opensearch')
     implementation 'org.apache.kafka:kafka-clients:3.4.0'
     implementation 'org.apache.avro:avro:1.11.0'
     implementation 'com.fasterxml.jackson.core:jackson-databind'
     implementation 'io.micrometer:micrometer-core'
     implementation 'org.apache.commons:commons-lang3:3.12.0'
+    implementation 'io.confluent:kafka-avro-serializer:7.3.3'
```

Comment:
shouldn't this be deserializer? or it's just name of the library that has both serializer and deserializer.